registry.json

```diff
@@ -5096,5 +5096,19 @@
    "telegram": "wafflepool",
    "operatorName": "",
    "github": ""
+ },
+ {
+   "ticker": "JED",
+   "poolId": "df2c28671a7d2e9b29703547533543926445f3a70f119a07af41d434",
+   "social": {
+     "twitter": "JStakepool",
+     "website": "https://adapool.jed.id.au",
+     "facebook": "",  
+     "instagram": "",
+     "youtube": ""
+   },
+   "telegram": "",
```

Comment:
You must provide a telegram username of an individual and not a group or channel